mystyle.css

```diff
@@ -0,0 +1,155 @@
+<style>
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding-top: 3.5rem;
+}
+
+.logo{
+   width:30%;
+   height:20%;
+}
+.slogan{
+  position: relative;
+  color: #45a049;
+  /* top: 200px;
+  left: 80px; */
+  text-align: left;
+}
+.header-image{
+  width: 500%;
+  heith: 900px;
+  background-color: blue;
+
+}
+a{
+  color: blue;
+}
+
+/* Style the header */
+.header {
+  overflow: hidden;
+  max-width:100%;
+  position:relative;
+  margin-left:auto;
+  margin-right:auto;
+  /* padding: 80px; */
+  text-align: center;
+  background: #04296e;
+  color: white;
+  padding-bottom: 30px;
+}
+.bgColor{
+    background-image: url('https://github.com/UWFront-End-Cert/adopt-a-dog-winter2020/blob/master/images/header-image.jpg?raw=true');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.liner-gradient-1{
```

Comment:
You have a typo here that prevents this from showing. But I'll give you half credit for this.